content/troubleshooting/build-performance.md

```diff
@@ -18,6 +18,149 @@ toc: true
 wip: true
 ---
 
-<!-- See https://discourse.gohugo.io/t/new-docs-site-need-feedback/5765/6-->
+{{% note %}}
+The example site used below is from https://github.com/gohugoio/hugo/examples/multilingual.
+{{% /note %}}
 
-<!-- Lots of forums threads that give the same answers. Should call attention to --verbose, --stepAnalysis, and --renderToMemory (for troubleshooting disk I/O). Highlight partialCached template function. -->
+
+## Step Analysis
```

Comment:
I like the idea and the rest of this page very much -- but I think the `stepAnalysis` tool is confusing more than it helps people. I suggest removing it (we could moving it to the bottom, but I have become a big fan of "less is more" lately).

Reply:
Thinking about it: I suggest move it to the bottom, but only mention it in a sentence, i.e. hot to invoke to get it, but don't list this intimidating table.

Reply:
I'll move it to the bottom and consider removing the sample output.  Thanks :+1: